### PR TITLE
Extract sendResponse method to it's own buildResponse class

### PR DESCRIPTION
**Pull request by ephieo**

### CODE REVIEW

File: src/test/java/BuildResponseTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @Test
+    public void checkThatHeadersLineIsParsed (){
+        BuildResponse build = new BuildResponse();
+        String headerKey = "Age";
+        String headerValue = "7777";
+
+        assertEquals(build.parseHeaderLine(headerKey,headerValue),"Age : 7777\n\r");
+
```
Comment (Kotauror) [inline src/test/java/BuildResponseTest.java:30]: I'm by no means Java expert and I'm aware that Java style requires you to add empty lines in places that are untypical for other languages like Python, ruby or JS. 

But the indentation in your file is not consistent - sometimes there are two empty lines between `assert` and `}`, sometimes one, make sure it's all aligned according to one standard.

File: src/main/java/http/response/BuildResponse.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package http.response;
+
+import java.util.*;
+
+public class BuildResponse {
```
Comment (Kotauror) [inline src/main/java/http/response/BuildResponse.java:5]: all of these functions make sense and are tested very well. I just wonder - what do you think about having one function - `buildResponse` that would use `buildStartLine`, `parseHeaderLine` and `buildHeaderLine` internally and these 3 will become private?

**Review by Kotauror — APPROVED** [review]
Very very good PR! 
Short, the title matches the content, it's all tested and written in a simple code 👏

File: src/main/java/http/response/BuildResponse.java (hunk 2 above)
Comment (ephieo) [reply src/main/java/http/response/BuildResponse.java:5]: Thanks justyna, I agree with this I'll make the change 👍

File: src/test/java/BuildResponseTest.java (hunk 1 above)
Comment (ephieo) [reply src/test/java/BuildResponseTest.java:30]: I'll check the indentation, InteliJ does the formatting for me but i'll double check.

**Pull request merged**